Formula/Kustomize.rb

```diff
@@ -0,0 +1,34 @@
+# WARNING: This file is auto-generated by formula.py.
+# Manual changes to this file will be overwritten the next time the file is generated.
```

Comment:
Is this for making the beta9 repo easier to install?

Reply:
I was planning on testing installing this from brew.